src/core/tsi/transport_security.h

```diff
@@ -91,6 +91,8 @@ struct tsi_handshaker {
 */
 struct tsi_handshaker_result_vtable {
   tsi_result (*extract_peer)(const tsi_handshaker_result* self, tsi_peer* peer);
+  tsi_frame_protector_type (*frame_protector_type)(
```

Comment:
Should we add a comment here (or somewhere else) saying a user must implement this API?